### ISSUE TITLE
Fix docs referencing twisted.web.request.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5887]
|<img alt="faulkner's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| faulkner reported|
|-|-|
|Trac ID|trac#5887|
|Type|defect|
|Created|2012-08-19 20:41:29Z|

Looks like this was moved to twisted.web.http.

Attachments:

* [5887.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/378/3787fb38fc27e6db5eb26d408586e2100abe52c9/8dd33271da2b7d48f3e3e52b7b8eb381ca2a0bc8.patch) (704 bytes) - added by faulkner on 2012-08-19 20:42:16Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5887 5887
type__defect defect
reporter__faulkner faulkner
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1345408889000000 1345408889000000
changetime__1345571693000000 1345571693000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks, looks good.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [35328]) Apply 5887.patch: Fix broken API link to t.web.http.Request in the templates howto.

Author: faulkner
Reviewer: thijs
Fixes: [#5887](https://github.com/twisted/twisted/issues/5887)
